util.go

```diff
@@ -43,3 +45,36 @@ func IsKeyInDomain(key, start, end []byte) bool {
 	}
 	return true
 }
+
+func FileExists(filePath string) bool {
+	_, err := os.Stat(filePath)
+	return !os.IsNotExist(err)
+}
+
+const (
+	strChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" // 62 characters
+)
+
+// Str constructs a random alphanumeric string of given length.
+func RandStr(length int) string {
```

Comment:
Do we use this in tests? If yes, then we should unexport it and move into `test_helpers.go`

Reply:
will move this to test_helpers.go